samples/nrf_rpc/protocols_serialization/server/README.rst

```diff
@@ -54,13 +54,13 @@ User interface
 .. tabs::
    .. group-tab:: nRF52840 DK
 
-      Button 4:
+      Button 1:
          * When the ``log_rpc`` snippet is enabled: triggers a fatal error.
             This is used for testing the crash log feature.
```

Comment:
The first bullet point entry seems to be in bold (https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-18608/nrf/samples/nrf_rpc/protocols_serialization/server/README.html#user-interface). May be this could fix it.
```suggestion
      Button 1:
      
         * When the ``log_rpc`` snippet is enabled: triggers a fatal error.
            This is used for testing the crash log feature.
```

Reply:
I think it's actually original Line 59. There seems to be an extra space at the beginning.